### PR TITLE
NAS-131308 / 25.04 / First time accessing a Minio deployment the webui button launches portal is 0.0.0.0:30001

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Testing: see ticket.
I updated it for `App Info` card and `Workloads` card (I see there is as well 0.0.0.0 host ip)

Result (instead of 0.0.0.0 we have localhost locally):

https://github.com/user-attachments/assets/7d55932c-ab01-49dc-8dc5-296d51da2335

### CODE REVIEW

File: src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.ts [hunk 1]
```diff
@@ -153,4 +155,16 @@ export class AppInfoCardComponent {
...
   }
+
+  private getPortalLink(app: App, name = 'web_portal'): string {
+    const portalUrl = new URL(app.portals[name]);
+
+    if (portalUrl.hostname === '0.0.0.0') {
+      const {
+        port, pathname, search, hash,
+      } = portalUrl;
+      return `${this.window.location.protocol}//${this.window.location.hostname}:${port}${pathname}${search}${hash}`;
```
Comment (stavros-k) [inline src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.ts:166]:
```suggestion
      portalUrl.hostname = this.window.location.hostname;
```
Would this work?

Comment (AlexKarpov98) [reply src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.ts]: @stavros-k - is that OK to update Workloads card as well? (to not see `0.0.0.0` there as well?)

Comment (AlexKarpov98) [reply src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.ts]: The code is updated now to remove updated logic from `Workloads` card

**Review by bvasilenko — APPROVED** [review]
Works as expected, and looks good from the code perspective

**Pull request merged**